waspc/data/Generator/templates/sdk/wasp/client/config.ts

```diff
@@ -1,7 +1,7 @@
-{{={= =}=}}
 import { stripTrailingSlash } from '../universal/url.js'
+import { env } from './env.js'
 
-const apiUrl = stripTrailingSlash(import.meta.env.REACT_APP_API_URL) || '{= defaultServerUrl =}';
```

Comment:
`env` objects is `import.meta.env` but validated. The default values are also defined in the Zod schema.